src/pages/authorization/agent-authorization/index.mdx

```diff
@@ -0,0 +1,676 @@
+---
+weight: 20
+category: "authorization"
+subcategory: "agent-authorization"
+title: "Agent Authorization"
+tagline: "Going through authorization in practice."
+desc: "Learn how to authorize your app to send a sample request using our Messaging APIs. A step-by-step tutorial that guides you through the process."
+---
```

Comment:
You need to go through our other API documents (Agent Chat, Customer, COnfiguration, Reports, etc.) and check the Authorization section. There are links that might need to be updates, for example https://developers.livechat.com/docs/messaging/agent-chat-api#authorization

Reply:
Okay, I did not think about that. I am doing it now